### PR TITLE
fix: absolute Paths conf.WebDir

### PR DESCRIPTION
**Pull request by Fuchange**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [X] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bugfix
- Description

When I build apisix-dashboard and run :  `APISIX_API_WORKDIR=/usr/local/apisix-dashboard/ /usr/local/bin/manager-api` at /root , 

there is no erros and said : 
```
2020-12-15T17:35:09.269+0800    filter/request_id.go:41 /       {"status": 404, "host": "10.1.30.28:9000", "query": "", "requestId": "69920330-aec8-4d31-b2f6-374909b0b3ed", "latency": 0, "remoteIP": "10.1.30.99", "method": "GET", "errs": []}
2020-12-15T17:35:13.961+0800    filter/request_id.go:41 /       {"status": 404, "host": "10.1.30.28:9000", "query": "", "requestId": "164126fd-adc8-4b0a-a434-fa4bc79209ff", "latency": 0, "remoteIP": "10.1.30.99", "method": "GET", "errs": []}
2020-12-15T17:35:14.093+0800    filter/request_id.go:41 /       {"status": 404, "host": "10.1.30.28:9000", "query": "", "requestId": "f2d3d9d1-a9a6-4210-9d37-95c3c449f86f", "latency": 0, "remoteIP": "10.1.30.99", "method": "GET", "errs": []}
```

- How to fix?

That is because conf.WebDir is fixed "./html/" , it doesn't change with conf.WorkDir . If Im not in WorkDir, it will be wrong.

So I change the path "conf.WebDir" to "conf.WorkDir + conf.WebDir".

I'm sorry I didn't set up issuse in advance, because I think this PR is too small.
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by nic-chen** [comment]
@Fuchange 
Test case needed.
Please refer: https://github.com/apache/apisix-dashboard/blob/master/api/test/shell/cli_test.sh
Thanks.

**Comment by Fuchange** [comment]
> @Fuchange
> 需要测试用例。
> 请参考：[https](https://github.com/apache/apisix-dashboard/blob/master/api/test/shell/cli_test.sh) : [//github.com/apache/apisix-dashboard/blob/master/api/test/shell/cli_test.sh](https://github.com/apache/apisix-dashboard/blob/master/api/test/shell/cli_test.sh)
> 谢谢。

@nic-chen  done, thanks!

File: api/conf/conf.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ const (
 	EnvDEV   = "dev"
 	EnvLOCAL = "local"
 
-	WebDir = "./html"
+	WebDir = "/html"
```
Comment (membphis) [inline api/conf/conf.go:39]: It seems wrong @nic-chen

File: api/internal/route.go [hunk 2]
```diff
@@ -49,9 +49,9 @@ func SetUpRouter() *gin.Engine {
 	store := cookie.NewStore([]byte("secret"))
 	r.Use(sessions.Sessions("session", store))
 	r.Use(filter.CORS(), filter.RequestId(), filter.RequestLogHandler(logger), filter.SchemaCheck(), filter.Authentication(), filter.RecoverHandler())
-	r.Use(static.Serve("/", static.LocalFile(conf.WebDir, false)))
+	r.Use(static.Serve("/", static.LocalFile(conf.WorkDir + conf.WebDir, false)))
```
Comment (tokers) [inline api/internal/route.go:52]: It's better to use `filepath.Join` here.

File: api/internal/route.go [hunk 3]
```diff
@@ -49,9 +49,9 @@ func SetUpRouter() *gin.Engine {
...
 	r.NoRoute(func(c *gin.Context) {
-		c.File(fmt.Sprintf("%s/index.html", conf.WebDir))
+		c.File(fmt.Sprintf("%s/index.html", conf.WorkDir + conf.WebDir))
```
Comment (tokers) [inline api/internal/route.go:54]: Ditto.

File: api/internal/route.go (hunk 2 above)
Comment (Fuchange) [reply api/internal/route.go:52]: ok, I will adjust it later.
 Thanks !

File: api/conf/conf.go (hunk 1 above)
Comment (nic-chen) [reply api/conf/conf.go:39]: yes, it should be `html/`

Comment (Fuchange) [reply api/conf/conf.go]: @membphis  @nic-chen  Done, please have a look~

File: api/internal/route.go (hunk 2 above)
Comment (Fuchange) [reply api/internal/route.go]: @tokers  Done, please have a look~

File: api/test/shell/cli_test.sh [hunk 4]
```diff
@@ -95,6 +95,26 @@ if [[ `grep -c "INFO" ./error.log` -eq '0' ]]; then
...
+mkdir html
+cd html
+echo "hi~" >> index.html
+APISIX_API_WORKDIR=$workDir $workDir/manager-api &
+sleep 5
+
+res=$(curl http://127.0.0.1:9000)
+pkill -f manager-api
+
+if [[ $res == "404 page not found" ]]; then
```
Comment (ShiningRush) [inline api/test/shell/cli_test.sh:110]: Here we should check the response is expected( `hi~` ), `404 page not found` just mean no resource.

Comment (Fuchange) [reply api/test/shell/cli_test.sh]: @ShiningRush done, thanks ~

**Review by ShiningRush — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]
LGTM

**Pull request merged**